cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/ArmInstanceConnector.java

```diff
@@ -0,0 +1,95 @@
+package com.sequenceiq.cloudbreak.cloud.arm;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloud.azure.client.AzureRMClient;
+import com.sequenceiq.cloudbreak.cloud.InstanceConnector;
+import com.sequenceiq.cloudbreak.cloud.arm.status.ArmInstanceStatus;
+import com.sequenceiq.cloudbreak.cloud.event.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.CloudVmInstanceStatus;
+import com.sequenceiq.cloudbreak.cloud.model.InstanceStatus;
+import com.sequenceiq.cloudbreak.cloud.model.InstanceTemplate;
+
+@Service
+public class ArmInstanceConnector implements InstanceConnector {
+
+    @Inject
+    private ArmClient armClient;
+
+    @Inject
+    private ArmMetadataCollector armMetadataCollector;
+
+    @Override
+    public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudInstance vm) {
+        return "-----END SSH HOST KEY FINGERPRINTS-----";
```

Comment:
Is it intentional? If yes at least a comment should be added why is this needed.